var/spack/repos/builtin/packages/py-voxcell/package.py

```diff
@@ -12,7 +12,7 @@ class PyVoxcell(PythonPackage):
     homepage = "https://bbpgitlab.epfl.ch/nse/voxcell"
     git      = "git@bbpgitlab.epfl.ch:nse/voxcell.git"
 
-    version('develop', branch='master')
+    version('develop', branch='main')
```

Comment:
IIRC, if we drop this, it should take the default branch?

Reply:
ahem, I already forgot about that trick.